packages/server/graphql/types/SetPokerSpectatePayload.ts

```diff
@@ -21,6 +25,24 @@ export const SetPokerSpectateSuccess = new GraphQLObjectType<any, GQLContext>({
         const meetingMember = await dataLoader.get('meetingMembers').load(meetingMemberId)
         return meetingMember
       }
+    },
+    stages: {
```

Comment:
+1 I'd call this `updatedStages` (because I didn't read the description & just assumed this was all stages)